applications/nrf_desktop/doc/hid_forward.rst

```diff
@@ -39,6 +39,8 @@ Complete the following steps to configure the module:
 #. The |hid_forward| is enabled with the :kconfig:`CONFIG_DESKTOP_HID_FORWARD_ENABLE` option.
    This option is available only if you enable both the :kconfig:`CONFIG_BT_CENTRAL` and :kconfig:`CONFIG_BT_HOGP` Kconfig options.
    The module is used on Bluetooth Central to forward the HID reports that are received by the HID service client.
+#. To enable forwarding of boot protocol reports enable options: :kconfig:`CONFIG_USB_HID_BOOT_PROTOCOL`, :kconfig:`CONFIG_DESKTOP_HID_BOOT_INTERFACE_KEYBOARD`
```

Comment:
I would prefer do avoid adding references to Kconfig options related to USB in this module (apart from CONFIG_USB_HID_DEVICE_COUNT that affects the module directly)

Reply:
I think we should either:
- refer to USB state module that already describes relevant USB-related options; BOOT_INTERFACE options can be described in hid_state (peripherals) and hid_forward (centrals)
- Introduce separated, more general section relate to HID boot protocol configuration in main doc page (if you think this is actually needed)

Reply:
btw. boot protocol code can be either 1 or 2, which means respectively boot keyboard or boot mouse